Phonebook.Frontend/src/app/shared/components/user/user-detail/user-detail.component.html

```diff
@@ -233,15 +233,15 @@ <h4 mat-line>
               </a>
             </h4>
             <p mat-line>
-              <li *ngFor="let shortOrgUnit of person.Business.ShortOrgUnit; let index = index">
+              <mat-chip-list class="mat-chip-list-stacked">

```

Comment:
Where is the class `mat-chip-list-stacked` defined?

Reply:
it comes with material

Reply:
Where can i find this classes? Didn't saw it here https://material.angular.io/components/chips/api

Reply:
Right there: https://material.angular.io/components/chips/overview#orientation